stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/application/App.kt

```diff
@@ -37,7 +37,7 @@ class App : Application() {
         super.onCreate()
         chatInitializer.init(getApiKey())
         instance = this
-        DebugMetricsHelper.init()
+        // DebugMetricsHelper.init()
```

Comment:
Out of curiosity, why do we need to remove this?

Reply:
Ops, my mistake 🤦